firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -367,9 +367,13 @@ class BrowserViewController: UIViewController,
         let showWarningBadge = consume isActionNeeded
 
         if isToolbarRefactorEnabled {
-            let badgeImageName = showWarningBadge ? StandardImageIdentifiers.Large.warningFill : nil
+            let (badgeImageName, maskImageName) = showWarningBadge ?
+            (StandardImageIdentifiers.Large.warningFill, ImageIdentifiers.menuWarningMask) :
+            (nil, nil)
```

Comment:
Just a nit because I don't like tuples and because this code is shorter:
```suggestion
            let badgeImageName = showWarningBadge ? StandardImageIdentifiers.Large.warningFill : nil
            let maskImageName = showWarningBadge ? ImageIdentifiers.menuWarningMask : nil
```